### PR TITLE
Bump actions/core to 1.10.0

### PR DESCRIPTION
**Pull request by stuartleeks**
Bump actions/core to pick up set-state replacement

Addresses part of #180

### CODE REVIEW

File: .github/workflows/ci_common.yml [hunk 1]
```diff
@@ -991,7 +991,7 @@ jobs:
           ref: ${{ inputs.prRef }}
 
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v1 
+        uses: docker/login-action@v2
```
Comment (stephenandary) [inline .github/workflows/ci_common.yml:994]: I am not sure why this is V1 for you, I have V2

File: github-action/package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.9.1",
+    "@actions/core": "^1.10.0",
```
Comment (stephenandary) [inline github-action/package.json:30]: I have bumped this in my branch because I need it to test

File: .github/workflows/ci_common.yml (hunk 1 above)
Comment (stuartleeks) [reply .github/workflows/ci_common.yml:994]: There's still one line with v1: https://github.com/devcontainers/ci/blob/main/.github/workflows/ci_common.yml#L994

**Comment by stuartleeks** [comment]
Build from branch (to test workflow changes): https://github.com/devcontainers/ci/blob/main/.github/workflows/ci_common.yml#L994

**Comment by stuartleeks** [comment]
Closing as I think all changes are now in #219

**Pull request closed without merging**